### PR TITLE
Scala Steward PRs 5650 - 5659 [BA-6518]

### PR DESCRIPTION
**Pull request by mcovarr**
~Do not review unless Chris's mega-PR proves unworkable. Does not include `workbench-google` as that was not a drop-in replacement.~
Review away!

### CODE REVIEW

File: project/Dependencies.scala [hunk 1]
```diff
@@ -60,14 +60,14 @@ object Dependencies {
   private val logbackV = "1.2.3"
   private val lz4JavaV = "1.7.1"
   private val mariadbV = "2.4.2"
-  private val metrics3ScalaV = "3.5.10" // https://github.com/erikvanoosten/metrics-scala/tree/f733e26#download-4x
+  private val metrics3ScalaV = "4.0.0" // https://github.com/erikvanoosten/metrics-scala/tree/f733e26#download-4x
```
Comment (cjllanwarne) [inline project/Dependencies.scala:63]: Comment potentially out of date?

**Comment by cjllanwarne** [comment]
Ping @grsterin for review

File: services/src/test/scala/cromwell/services/database/LobSpec.scala [hunk 2]
```diff
@@ -61,19 +61,7 @@ class LobSpec extends FlatSpec with Matchers with ScalaFutures {
...
-            exception should be(a[SerialException])
-            exception.getMessage should
-              be("Invalid arguments: position cannot be less than 1 or greater than the length of the SerialBlob")
-          }
-        case _ =>
-          database.addWorkflowStoreEntries(workflowStoreEntries)
-      }
-
-      future.futureValue
+      database.addWorkflowStoreEntries(workflowStoreEntries).futureValue
```
Comment (grsterin) [inline services/src/test/scala/cromwell/services/database/LobSpec.scala:64]: Why don't we need special handling for MySql anymore?

Comment (mcovarr) [reply services/src/test/scala/cromwell/services/database/LobSpec.scala:64]: presumably because the upgraded dependency corrected the problem.

**Pull request merged**